### ISSUE TITLE
Not able to replay KeyUp and KeyDown step

### ISSUE THREAD

**Issue by jecfish** [webdriverio/recorder-extension#4]
Steps:

1. Import this JSON file in Recorder. https://github.com/puppeteer/replay/issues/268
2. Export it as WebDriverIO script.
3. Observe the step `{ "type": "keyDown","key": "Escape"}` is transform to 
    ```
     await browser.performActions([{
      type: 'key',
      id: 'keyboard',
      actions: [{ type: 'keyDown', value: 'ESCAPE' }] 
    }])
   ```
5. Run it with chromedriver
6. Receive error `invalid argument: 'value' must be a single Unicode code point` [NOT OK]
7. Change the value to `\x1B`. Step executed successfully.

Expect the keyboard action map to unicode correctly.

Notes:

When the error happen, I tried to search for relevant official documentation, no useful info found.

Search online randomly return some suggests to add a config. Doesn't work either. 

```
'goog:chromeOptions': {
            w3c: true,
        },
```

**Comment by christian-bromann** [comment]
Thanks for raising the issue.

Sending `ESCAPE` is not quite correct here. WebdriverIO has [a table](https://github.com/webdriverio/webdriverio/blob/6bbe1cf371f06e517ac62a4e02724c8b610803cb/packages/wdio-utils/src/constants.ts#L5) to map these value to an unicode character.

**Comment by christian-bromann** [comment]
fixed and published as `v0.3.3`
